src/Orchard.Web/Modules/Orchard.Azure.MediaServices/Orchard.Azure.MediaServices.csproj

```diff
@@ -147,6 +147,8 @@
     <Reference Include="System.Xml.Linq" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Assets\Css\azuremediaplayer.min.css" />
+    <Content Include="Assets\JavaScript\Lib\azuremediaplayer.min.js" />
```

Comment:
Correction: minified versions should be handled by gulp, I agree, though they will still be committed in the repos.
I will try to find the non-minified version for these files.